packages/grafana-ui/src/components/FilterPill/FilterPill.tsx

```diff
@@ -39,7 +39,6 @@ const getFilterPillStyles = stylesFactory((theme: GrafanaTheme, isSelected: bool
       padding: ${theme.spacing.xxs} ${theme.spacing.sm};
       background: ${theme.colors.bg2};
       border-radius: ${theme.border.radius.sm};
-      display: inline-block;
```

Comment:
Could this cause some use cases to render each `FilterPill` on a new line or are they always wrapped in a `HorizontalGroup` or similar?

Reply:
This was a duplicate declaration, there's `display: flex` rule below, so I removed this one as it is overwritten anyway.